web/src/features/fba/components/map/ScaleBarContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from '@mui/material'
+import React, { forwardRef } from 'react'
+
+const ScalebarContainer = forwardRef((_, ref) => {
+  return (
+    <Box
+      data-testid="scalebar-container"
+      sx={{
+        backgroundColor: '#FFFFFF',
+        border: '1px solid #000000',
+        padding: '12px',
+        position: 'absolute',
+        right: '10px',
+        bottom: '10px',
+        overflow: 'hidden'
+      }}
+    >
+      <Box
+        ref={ref}
+        sx={{
+          ['& div.ol-scale-bar']: {
+            position: 'static'
+          },
+          ['& div.ol-scale-bar-inner div.ol-scale-step-marker']: {
+            top: '13px !important'
```

Comment:
The `!important`s are ugly but necessary to override the OpenLayers styles which are declared on the actual HTML elements, not just in css classes.